tests/graph/test_config.py

```diff
@@ -403,6 +407,7 @@ def test_generate_object_field_with_data_categories(self):
                 is_array=False,
                 sub_fields=[apt_no_sub_field],
                 return_all_elements=None,
+                read_only=False
```

Comment:
We could add a test where read_only=True for completeness and assert that the created field has this propagated.

Reply:
Test added!